tests/conftest.py

```diff
@@ -406,6 +413,33 @@ def remove_data_written_flag():
 
 
 @pytest.fixture
-def no_numbagg():
-    with xr.set_options(use_numbagg=False):
-        yield
+def swe_series():
+    def _swe_series(values, start="1/1/2000", units="mm"):
+        coords = pd.date_range(start, periods=len(values), freq="D")
+        return xr.DataArray(
+            values,
+            coords=[coords],
+            dims="time",
+            name="swe",
+            attrs={
+                "standard_name": "snow_water_equivalent_in_snow_layer",
```

Comment:
This standard name doesn't exist : https://cfconventions.org/Data/cf-standard-names/current/build/cf-standard-name-table.html